src/sass/base/_container.scss

```diff
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  min-width: 320px;
+  margin: 0 auto;
+  padding-left: 20px;
+  padding-right: 20px;
+  align-items: center;
+}
+@media screen and (min-width: 480px) {
```

Comment:
w media nie musisz określać max width dla contenera gdyż określenie max-width nadaje maksymalną szerokość ekran który będzie miał 300px szerokości to contener bedzie miał 300- padding czyli 260px dalej mieście się w przedziale max-with 1280px

Reply:
OK